app/assets/stylesheets/_layout.scss

```diff
@@ -0,0 +1,19 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+figure {
```

Comment:
I’ve started using it more and more because it gives you the `figcaption` element, which can be used to describe an image (for example) in more detail. This improves accessibility, but the `figcaption` can also be visually hidden so to not be in the actual layout while still giving something for screen readers to read.